src/chess/piece/Pawn.java

```diff
@@ -1,24 +1,81 @@
 package chess.piece;
 
-import java.util.HashMap;
-import java.util.Map;
+import chess.Position;
+
+import java.util.*;
 
 public class Pawn extends Piece {
   private static final Map<Color, Character> symbols = new HashMap<Color, Character>(){{
     put(Color.WHITE, '♙');
     put(Color.BLACK, '♟');
   }};
 
+  // if this pawn is moved, isFirstMoved turn out true but at next your own turn, it turn out false
+  // this variable is related to "en passant"
+  // ex. n:false -> n + 1:true -> n + 2:false
+  private boolean isFirstMoved;
+
+  // if this pawn is moved, isMoved turn out true and it never change again
+  // when the value is true, this Pawn can move forward two squares.
+  // ex. n:false -> n + 1:true
+  private boolean isMoved;
```

Comment:
Do you think this is an irregular parameter of Pawn only?
If so, here is the right place to put, but if this parameter should be held by all pieces (which means all pieces potentially change their behavior depending on they have moved or not), you should put this field to the parent class